x-pack/plugins/siem/common/endpoint_alerts/types.ts

```diff
@@ -15,28 +15,9 @@ import {
   AlertEvent,
   KbnConfigSchemaInputTypeOf,
   AppLocation,
+  Immutable,
 } from '../endpoint/types';
 
-/**
```

Comment:
This is a second copy of these types. I copy-pasted it back when I thought the subplugins would be separate from each other.